libs/localization/src/lib/en.json

```diff
@@ -1070,6 +1070,7 @@
       "infoTitle": "Additional information on aggregate feature importance",
       "legendHelpText": "Toggle cohorts on and off in the plot by clicking on the legend items.",
       "missingParameters": "This tab requires the local feature importance parameter be supplied.",
+      "sortByCohort": "Sort by cohort",
       "sortBy": "Sort by datapoint",
```

Comment:
is sortBy used anywhere else? If not, can we remove it

Reply:
Yes, it is used in local importance chart.